src/store/token.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+export async function getToken(token) {
+	const tokenValue = await ((typeof (token) === 'function')
+		? token()
+		: token);
+
+	return await new Token(tokenValue);
+}
+
+export function compareTokens(a, b) {
+	return a.toString() === b.toString();
```

Comment:
What's the use case for comparing tokens?

Reply:
To see if a new token is just a refreshed version of an old token.